lib/adroll/demo_responses.rb

```diff
@@ -0,0 +1,705 @@
+require 'json'
+
+module DemoResponses
+    def make_demo_response(request_method, request_uri, params, additional_query_params)
+        status, data, body_string = demo_response_default
+
+        case request_uri
```

Comment:
Should there be a default case for this?

Reply:
I was going to do that at first, but ended up putting in the above line instead (`status, data, body_string = demo_response_default`). That's because the actions in the case statement also check the request_method, so the default case would miss some of them, like if the first case matches but the request method is not put.

Reply:
Ah ok. Just wanted to make sure it could not be nil going into a regex